src/components/organisms/AdminVenueView/components/RunTabUserInfo/RunTabUserInfo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+import { WithId } from "utils/id";
+
+import { User } from "types/User";
+
+import { useWorldUserLocation } from "hooks/users";
+
+export interface RunTabUserInfoProps {
+  user: WithId<User>;
+}
+
+export const RunTabUserInfo: React.FC<RunTabUserInfoProps> = ({ user }) => {
+  const { userLocation } = useWorldUserLocation(user?.id);
+  const userLastSeenIn = userLocation?.lastSeenIn;
+
+  const location = userLastSeenIn ? Object.keys(userLastSeenIn)?.[0] : "";
+
+  return (
+    <div key={user.id} className="RunTabUsers__row RunTabUsers__user">
+      <UserAvatar user={user} showStatus />
```

Comment:
```suggestion
      <UserAvatar user={user} showStatus size="small" />
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1705/commits/f038fa7f07dd402677ba46b787847df727270978